sklearn/utils/testing.py

```diff
@@ -406,7 +406,7 @@ def assert_raise_message(exceptions, message, function, *args, **kwargs):
     exceptions : exception or tuple of exception
         Name of the estimator
 
-    func : callable
+    function : callable
```

Comment:
Could you rename the param `function` to `func` instead to conform to [existing nomenclature](https://github.com/scikit-learn/scikit-learn/pull/6700/files#diff-f0af7b240082b7b0e03078a38121ae1aL192) instead?

You have to deprecate it though as utils is still considered public.